src/classes/PMT_PaymentWizard_TEST.cls

```diff
@@ -351,6 +362,7 @@ private class PMT_PaymentWizard_TEST {
             stagename = closedWonStage()
         );
         insert opp1;
+        list<opportunity> queryopp = [select id, currencyisocode from opportunity where id=:opp1.id];
```

Comment:
why add this query to the test?  It isn't used anywhere that I can see.  should it?  maybe some assert dealing with currencyisocode?